guides/v2.1/test/integration/integration_test_execution.md

```diff
@@ -21,37 +21,52 @@ functional_areas:
 Integration tests require the Magento runtime environment, so they need a little preparation before they can be executed.
 Once the system is prepared, the tests can be executed using either the command line interface or within an IDE like PhpStorm.
 
-### Set up the integration test framework
+## Set up the integration test framework
 
 In order to run the integration tests, create and configure a test database.
 You might also want to adjust the PHPUnit configuration, depending on your requirements.
 
 Refer to [Preparing Integration Test Execution][setup] for further information about setting up the test environment.
 
-### Command Line Interface (CLI)
+## Command Line Interface (CLI)
 
 This option can be used for running the tests locally during development or on remote servers during Continuous Integration.
 
 Please refer to [Running Integration Tests in the CLI][cli run] for further information.
 
-### PhpStorm IDE
+## PhpStorm IDE
 
 Running the integration tests inside an IDE like PhpStorm IDE is convenient during development. This is mostly used when writing a new integration test.
 
 Other than convenience, there is no benefit over running the tests on the console.
 
 Refer to [Running Integration Tests in PhpStorm][phpstorm run] for further information.
 
+## Integration tests file structure
+
+The root folder for the Magento integration tests suite is `<magento_root>/dev/tests/integration`.
+
+This folder contains the following sub-folders and files:
+
+- `framework/` – Integration testing framework scripts, configuration files and classes.
+- `Magento/` – A set of classes that actually implement the Magento integration tests framework.
+- `bootstrap.php` – The PHPUnit bootstrap script.
+- `etc/install-config-<db_vendor>.php` – A configuration file that provides values for installing Magento application.
+- `testsuite/` – The test suite.
+- `tmp/` – A writable directory for storing temporary data during tests execution.
```

Comment:
This line just sounds funny to me right now. "during tests execution"
Not sure whether it is the word execution or if it is the plural tests. Reread just to be sure.

Reply:
Right. Made singular.